man/systemd.exec.xml

```diff
@@ -1321,6 +1321,19 @@ BindReadOnlyPaths=/var/lib/systemd</programlisting>
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ProtectKernelLogs=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, access to the kernel ring buffer will be denied. It is
+        recommended to turn this on for most services that do not need to read kernel logs. Enabling this option
+        removes <constant>CAP_SYSLOG</constant> from the capability bounding set for this unit, and installs a system
+        call filter to block the <citerefentry project='man-pages'><refentrytitle>syslog</refentrytitle><manvolnum>2</manvolnum></citerefentry>
+        system call. The kernel exposes logs to userspace via <filename>/dev/kmsg</filename> and
+        <filename>/proc/kmsg</filename>. If enabled, these are made inaccessible to all the processes in the unit.</para>
```

Comment:
exposes logs → exposes its log buffer

(background: the logs are eventually available through other means because journald for example write them to disk. hence, this option restricts access to the log buffer not the logs so much, since one is an interface the other just a concept of information, if you follow what i mean. — i do agree though that the whole option should still be named ProtectKernelLogs= [and not ProtectKernelLogBuffer=] since this distinction is good to make in prose but appears a bit too philosophical to expose in the option, which should be named brief I think)